Array.qml

```diff
@@ -36,6 +36,10 @@ Entity {
                             : repeatedEntity.index == root.lastIndex && lastItemDelegate
                               ? lastItemDelegate
                               : itemDelegate
+
+                Component.onCompleted: {
+                    object.index2 = index;
```

Comment:
Perhaps a more general workaround would be `if(object.hasOwnProperty("index2")) object.index2 = repeatedEntity.index` (why index2 btw?)